### PR TITLE
Split README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

**Comment by clementbiron** [comment]
@martinratinaud according to #54, could you please document all environement variables ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Markdown Guidlines
+
+Although we use the MDX format which sometimes allows the instantiation of React components in mdx files (see the [Readme section on this subject](/README.md#MDX)), it is recommended to avoid doing so to facilitate the contribution.
+
+Prefer Markdown over strings for translations, as soon as there are multiple paragraphs.
+
+## Local creation of services from contribution interface
+
+If you are interested in setting up a local instance where you can directly save the result of the contribution interface (`/contribute`) in the corresponding folder in Open Terms Archive project you can add the following constant to your `.env.local`
```
Comment (Ndpnt) [inline CONTRIBUTING.md:11]: If you are interested in setting up a local instance where you can directly save the result of the contribution interface (`/contribute`) in the corresponding folder in Open Terms Archive project you have to specify the path with the following constant in `.env.local` file:

File: README.md [hunk 2]
```diff
@@ -1,68 +1,81 @@
...
+Clicking on `Validate` can automatically create an issue un a github repository.
 
-Get your local IP
+For this, you need to setup some constants in your `.env.local`
 
 ```
-ipconfig getifaddr en0
+GITHUB_TOKEN_CREATE_ISSUE= #create one with repo privileges here https://github.com/settings/tokens
+GITHUB_OTA_OWNER=ambanum
+GITHUB_OTA_REPO=test-repo# or OpenTermsArchive
```
Comment (Ndpnt) [inline README.md:28]: Avoid `test-repo`, directly `GITHUB_OTA_REPO=OpenTermsArchive`

**Review by Ndpnt — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (martinratinaud) [reply README.md]: In fact i will change this to the same env variables as on OTA Core so you can leave it

**Review by martinratinaud — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# CONTRIBUTING
+
+## Markdown Guidlines
```
Comment (martinratinaud) [inline CONTRIBUTING.md:3]: Guidelines

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Although we use the MDX format which sometimes allows the instantiation of React components in mdx files (see the [Readme section on this subject](/README.md#MDX)), it is recommended to avoid doing so to facilitate the contribution.
```
Comment (martinratinaud) [inline CONTRIBUTING.md:5]: ./README I think

**Review by martinratinaud — APPROVED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (martinratinaud) [reply CONTRIBUTING.md:5]: I meant `./README.md#MDX` instead of `/README.md#MDX`. with the leading `.`

Comment (clementbiron) [reply CONTRIBUTING.md:5]: it's working without the `.`

Comment (martinratinaud) [reply CONTRIBUTING.md:5]: ok sorry then

**Pull request merged**